stonesoup/predictor/kalman.py

```diff
@@ -70,7 +70,7 @@ def _transition_matrix(self, **kwargs):
         Returns
         -------
         : :class:`numpy.ndarray`
-            The transition matrix, :math:`F_k`
+            The transition matrix, :meth:`F_k`
```

Comment:
These should be `:math:` still.

Reply:
Confused myself. I've just rebased the entire file back off master since I didn't need to change any of this.